### ISSUE TITLE
Missing package/packages?

### ISSUE THREAD

**Issue by bgruening** [bioarchive/aRchive_source_code#16]
For example this one seems to be missing: http://bioconductor.org/packages/3.1/data/experiment/src/contrib/HSMMSingleCell_0.102.0.tar.gz

**Comment by nturaga** [comment]
@bgruening This package doesn't seem to be present in the SVN repo I got from "hedgehog" which is public 'https://hedgehog.fhcrc.org/bioconductor/branches/RELEASE_3_0/madman/Rpacks/'. On both release 3.0 and 3.1 of BioC and I'm not sure why. But i'll look into it today.

**Comment by nturaga** [comment]
Actually, that seems to be just experiment data, and it wouldn't be available on hedgehog.

**Comment by bgruening** [comment]
@nitesh1989 yes these are experimental data, but it shows up as dependency. I need to figure out if this is really a hard dependency.
